angular/src/app/containers/form/form.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  OnInit
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+} from '@angular/forms';
+import { Store } from '@ngrx/store';
+import * as reducers from '../../reducers';
+import { SetSelectionAction } from '../../actions/map';
+
+@Component({
+  selector: 'dp-form',
```

Comment:
Ja inderdaad. Ik zit nu te denken aan `dp-data-controls` omdat er meer controls bij zouden kunnen komen en de functie is om te configureren hoe/welke data er terug komt.